### DISCUSSION TITLE
Server 504s on first load after clearing the cache

### DISCUSSION THREAD

**Discussion by johncarter-** [statamic/cms#7771] [Troubleshooting]
When I clear my site's cache `php artisan optimize:clear` it takes the site offline for about 30 secs while I assume the stache is being warmed. Is that to be expected, if not, how do I avoid it?

I'm on a Forge provisioned, 1GB DO droplet with no other sites on it and it reaches 96% memory usage for that period.

**Comment by joshuablum** [comment]
Hey @johncarter-,

Is the [Stache Watcher](https://statamic.dev/stache#watcher) enabled on the server?

Also, if you run the individual commands that `artisan optimize` summarizes (like views, routes, events, config) `artisan cache` is the one causing the issue?

**Comment by johncarter-** [comment]
Nope. That's `false`. Thinking back, I think this might be a "number of assets" issue, but I can't be sure — will check. The site has a 15000+ assets.
